lib/spack/spack/util/compression.py

```diff
@@ -168,6 +188,10 @@ def _unzip(archive_file):
 
 
 def _unZ(archive_file):
```

Comment:
(minor) I think for clarity it might be worth calling this `_system_unZ`: I think if that were done, the naming conventions would be the following:

* `_system_x` uses a system utility
* `_py_x` uses builtin python support
* `_x` could use either